### ISSUE TITLE
[HttpTrigger] Receiving 403 when not sending Expect: 100-continue

### ISSUE THREAD

**Issue by Ultre00** [Azure/azure-functions-dotnet-worker#1005]
We are hosting an Azure Function V4 with runtime type dotnet-isolated.

The package versions are:
Microsoft.Azure.Functions.Worker "1.6.0"
Microsoft.Azure.Functions.Worker.Extensions.Http "3.0.13"

When the Function is deployed to Azure we are facing some issues with larger requests where the request body is over let's say 100 kb. We immediately get a 403 error. This doesn't even log a requests so we don't have any logging we can view. The only way we get it to work is by adding a Expect: 100-continue header. However we don't expect our users to always send this. How can we disable this feature?

**Comment by kshyju** [comment]
@Ultre00 Is this a V3 app or V4? You can check the `AzureFunctionsVersion` element value in your .csproj file. Do you have a minimal repro you could share? (A github repo will be great)

**Comment by Ultre00** [comment]
@kshyju as mentioned this is a V4 function. I also figured out when this happens. It is only in combination with mTLS where the option "clientCertEnabled" is set to true. For now I simply set this to false since we are no longer using mTLS here.

**Comment by brettsam** [comment]
@Ultre00 -- where did you have these configurations set? Is this somewhere in App Service?

**Comment by Ultre00** [comment]
@brettsam yes you can set that in the Function App. If you use an ARM template the properties are called
`clientCertEnabled` in combination with `clientCertMode` but if you use the UI then you can also do this under configuration > general settings > Client certificate mode
